language/tools/move-cli/src/package/cli.rs

```diff
@@ -86,7 +139,10 @@ pub enum PackageCommand {
     },
     /// Build the package at `path`. If no path is provided defaults to current directory.
     #[clap(name = "build")]
-    Build,
+    Build {
+        #[clap(long = "arch", parse(try_from_str = Architecture::try_parse_from_str))]
```

Comment:
So there is already a new option to the package system and compiler as seen here: https://github.com/diem/move/blob/main/language/extensions/async/examples/account/Move.toml#L11.

It currently can have values "async" and "evm". This is passed into the compiler in order to enable attribute processing. For EVM, for example, `#[callable] fun f` leads to generation of native functions `#[extern] public native fun call_f` for cross-contract calls. I wonder whether we should try to  unify this.